docs/src/guides/working-with-projects.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Working with Projects
+navTitle: Working with Projects
+description: Working with Hub Projects
+order: 12
+group: 2-concepts
+---
+
+## Working with Projects
+
+### In an Application
+
+Assuming the following routes in an app...
+
+```js
+//...snip...
+this.route("projects", function () {
+  this.route("project", { path: "/:project_id" }, function () {
+    this.route("edit");
+  });
+  this.route("edit", function () {
+    this.route("new");
+  });
+});
+//...snip...
+```
+
+#### Create a Project
+
+To create a project in the `/projects/edit/new` controller...
+
+```js
+import Controller from "@ember/controller";
+import { inject as service } from "@ember/service";
+import { action } from "@ember/object";
+import { HubProjectManager } from "@esri/hub-common";
+
+export default class projectsEditNewController extends Controller {
+  @service appSettings;
+
+  @action
+  createProject(project) {
+    const projectMgr = HubProjects.create(appSettings.contextManager);
+    return projectMgr.create(params.project_id);
```

Comment:
I believe with recent change we should now be able to do something like
`const project = Hub.projects.create(project)`

is that correct?